### ISSUE TITLE
[Feat]: Allow editing assistant responses

### ISSUE THREAD

**Issue by dave1010** [a-ghorbani/pocketpal-ai#137]
**Description**

https://github.com/a-ghorbani/pocketpal-ai/pull/128 introduced the ability to edit user messages, which is great.

It would also be useful to be able to edit the model's responses. This would work in an identical way to editing user messages.

**Use Case**

* It's often much more effective to correct the model's own response, rather than adding a new user message.
* Editing the model's response also allows you to persuade the model much more too, as it thinks that it's already said something.
* Removing text from the model's response if it's too verbose can speed up inference.

**Comment by stradeleve** [comment]
Indeed, if we could just edit the responses from AI like how ChatterUI does, that would be great. The problem with ChatterUI is that the AI's responses aren't as good compared to this app. If we could edit the AI's responses, we wouldn't have to go to such lengths to make them understand what we want them to do. We could just tweak their responses a bit to align them with the right direction, and they'd handle the rest. This would make things much less of a hassle. Also, I hope there aren't any limits on their responses.

**Comment by TomerGamerTV** [comment]
Bump

**Comment by Foul-Tarnished** [comment]
Please implement this 🙏🏻

Small models can be a bit stubborn and repeat the same crap, editing their response can help a lot to "unlock" them

**Comment by nightkall** [comment]
**Editing and continue a message are an essential features**. You can steer the conversation editing the bot's message. 

For example you can do a simple jailbreak writing "Sure, here you have:" and the LLM will answer whatever you asked. 

Thank you for your app and your time.
